devops/jobs/ExportRDSSlowQueryLogs.groovy

```diff
@@ -0,0 +1,68 @@
+package devops.jobs
+import static org.edx.jenkins.dsl.Constants.common_wrappers
+
+class EXPORTRDSSLOWQUERYLOGS {
+    public static def job = { dslFactory, extraVars ->
+        assert extraVars.containsKey("DEPLOYMENTS") : "Please define DEPLOYMENTS. It should be list of strings."
+        assert !(extraVars.get("DEPLOYMENTS") instanceof String) : "Make sure DEPLOYMENTS is a list of string"
+
+        extraVars.get('DEPLOYMENTS').each { deployment ->
+
+                dslFactory.job(extraVars.get("FOLDER_NAME","Monitoring") + "/export-slow-query-logs-${deployment}") {
+                    parameters {
+                        stringParam('CONFIGURATION_REPO', 'https://github.com/edx/configuration.git')
+                        stringParam('CONFIGURATION_BRANCH', 'master')
+                    }
+
+                    wrappers common_wrappers
+
+                    wrappers {
+                        credentialsBinding {
+                            string("USERNAME", "${deployment}-export-slow-logs-username")
+                            string("PASSWORD", "${deployment}-export-slow-logs-password")
+                            file("AWS_CONFIG_FILE","tools-edx-jenkins-aws-credentials")
+                            def variable = "${deployment}-export-slow-query-logs"
+                            string("ROLE_ARN", variable)
+                        }
+                    }
+
+                    triggers {
+                        cron("H * * * *")
```

Comment:
Since your script looks back an hour in the logs, this should not run every minute.  It should probably run every 55 minutes or so.  This is because we want to minimize overlap while not losing anything.

Reply:
Let me know if you need more context on this